src-electron/db/query-command.js

```diff
@@ -724,6 +724,7 @@ async function selectAllCommandsWithArguments(db, packageId) {
       argFieldId: x.FIELD_IDENTIFIER,
       argIsArray: x.ARG_IS_ARRAY,
       argPresentIf: x.ARG_PRESENT_IF,
+      argIsNullable: x.ARG_IS_NULLABLE,
```

Comment:
Why do we not need `dbApi.fromDbBool` here?

Reply:
We do need it. We're not the most disciplined, because in all reality, the `1` and `0` that is the database boolean, automagically maps into JavaScript world just fine, because JavaScript was designed that way, but it makes me uncomfortable still, so I like to have `fromDbBool()` wherever I can.
I'll add it in.